contrib/scripts/make_user_admin_in_room.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+
+import json
+import sys
+import urllib
+from argparse import ArgumentParser
+
+import requests
+
+
+def _mkurl(template, kws):
+    for key in kws:
+        template = template.replace(key, kws[key])
+    return template
+
+
+def main(hs, room_id, access_token, user_id):
+    headers = {'Authorization': 'Bearer %s' % (access_token)}
+    room_state_url = _mkurl(
+        "$HS/_matrix/client/api/v1/rooms/$ROOM/state?access_token=$TOKEN",
+        {"$HS": hs, "$ROOM": room_id, "$TOKEN": access_token},
+    )
```

Comment:
I may ask a silly question, but why not using the `str.format()` function in python:

```python
"{HS}/_matrix/client/api/v1/rooms/{ROOM}/state?access_token={TOKEN}"\
  .format(HS = hs, ROOM = room, TOKEN = token)
```

So, if I am not wrong, you would not need `_mkurl` anymore.

Reply:
True, I'll edit this when I have time unless someone else wants to. I think it's six one way half a dozen the other considering this is just a simple administration script.

Reply:
Fixed: 
https://github.com/matrix-org/synapse/pull/10811/commits/98ee4bd417396c6f61f724b91a75955f9ebd5cce

Reply:
just for your own reference, this is something in Python 3.6+ as well:

```python
f"{hs}/_matrix/client/api/v1/rooms/{room}/state?access_token={token}"
```